lib/render-html.js

```diff
@@ -8,25 +10,24 @@ const tryResolve = (...args) => {
 
 export default async ({ resume, themePath }) => {
   const cwd = process.cwd();
-  let path;
+  let resolvedPath;
   if (themePath[0] === '.') {
-    path = tryResolve(path.join(cwd, themePath), { paths: [cwd] });
-    throw new Error(
```

Comment:
I don't really know why there was a throw here. It seems like it would always fail for a path that begins with a dot. I removed that to make it work.